gmsm/gmsm-htb-system-tests.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
```

Comment:
Please implement the following functions as outlined [here](https://kb.indexexchange.com/Adapters/Test_Cases.htm) to enable testing for more features:

- `getValidResponseWithDeal`
- `validateTargetingWithDeal`
- `validateBidRequestWithPrivacy`